locale/en/get-involved/node-meetups.md

```diff
@@ -0,0 +1,56 @@
+# Node Meetups
+
+This is a list of node meetups. Please submit a PR if you'd like to add your local group!
+
+## Code of Conduct
+
+If any meetup does not have a CoC and/or is reported as an unsafe place, it will be removed from this list. 
+
+## Notes for adding meetups
+
+FORMAT
+- [Meetup](https://www.meetup.com/pdxnode/)
+- Frequency of meetups 
+- How to submit a talk? <<list here>>
```

Comment:
I'd suggest a supplying a talk submission template (similar to [this](https://github.com/cambiecollective/vanjs/blob/master/.github/ISSUE_TEMPLATE/talk-idea.md) used by VanJS) that can be customized by meetups.

A template like that produces talk submissions like [this example](https://github.com/cambiecollective/vanjs/issues/8).

Reply:
@ladyleet yes, an example would be totally rad! I always get extra value from examples 👍🏽